tests/mock_certmonger.py

```diff
@@ -20,7 +23,11 @@
         'cert-storage': 'FILE',
         'cert-presave-command': template % 'renew_ra_cert_pre',
         'cert-postsave-command': template % 'renew_ra_cert',
-        'not-valid-after': 1024,
+        'not-valid-after': (
+            int(
+                datetime(1970, 1, 1, 0, 17, 4, tzinfo=timezone.utc).timestamp()
```

Comment:
A fancier way of doing it :-) I  think this was my way of forcing an guaranteed-expired cert but formalizing it is a good thing.